js/parsers/parser-input-select.js

```diff
@@ -226,6 +226,9 @@
 				this.value = $( this ).data( 'ts-original-value' );
 			})
 			.on( 'change keyup '.split( ' ' ).join( namespace + ' ' ), 'select, input, textarea', function( event ) {
+				if (this.classlist.classList.contains('parser-false')) {
```

Comment:
`classList.classList` should be `event.target.classList`? Either way, this is targeting form elements (`select`, `input` and `textarea`), so adding a `parser-false` to these elements won't do anything. Was this supposed to target the header?

Reply:
No it’s targeting a select within a cell to allow individual cells to request their input be ignored.

Reply:
This change allows users to ignore change events for specific cells and not the entire column.

Reply:
This event handler the way it is  interfering with drop down clicks from bootstrap drop downs initiated from input search. Requiring blueing focus then clicking on drop down menu items.

Reply:
Ok, thanks. That makes sense. I plan on making a bunch of updates to the code base this weekend, so I'll bump the version after.

Reply:
> Ok, thanks. That makes sense. I plan on making a bunch of updates to the code base this weekend, so I'll bump the version after.

I did take a look for a place to document, but couldn't figure where to put it.... is there a doc you think I should update?

Reply:
The input/select parser on the main doc page links to the group widget - https://mottie.github.io/tablesorter/docs/example-widget-grouping.html